torch/csrc/jit/codegen/cuda/parser.cpp

```diff
@@ -1830,42 +1833,51 @@ class IrParser {
     }
 
     {
-      auto ptr_op = getOperatorForLiteral(
-          "aten::softmax.int(Tensor self, int dim, int? dtype) -> Tensor");
-      REGISTER_PARSE_RULE(
-          ptr_op,
-          {
-            MemoryFormat format;
-            std::list<Val*> list_val;
-            std::tie(format, list_val) = getConsistentValues(
-                MemoryFormat::Contiguous(),
-                value_map[node->inputs()[0]->unique()]);
-            auto input_t = list_val.front();
-            list_val.pop_front();
-            auto input = input_t->as<TensorView>();
+      std::array<const char*, kNumSoftmaxFwd> SoftmaxFwd = {
+          "aten::softmax.int(Tensor self, int dim, int? dtype) -> Tensor",
```

Comment:
nitpick:
"aten::softmax.int(Tensor self, int dim, int? dtype) -> Tensor" could be updated to
"softmax.int(Tensor self, int dim, ScalarType? dtype=None) -> Tensor"

I never really understand their schema matching.... Magically here `int?` and `ScalarType?` resolves to the same thing, but sometimes argument names have to match exactly :shrug: